notebooks/default-prep-script.ipynb

```diff
@@ -284,7 +284,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.6"
+   "version": "3.11.2"
```

Comment:
Downgrade?

Reply:
it's just what jupyter does.  Per discussion leaving alone.